pocs/filterwheel/sbig.py

```diff
@@ -44,6 +44,9 @@ def __init__(self,
                          timeout=timeout,
                          serial_number=serial_number,
                          *args, **kwargs)
+        # All SBIG filterwheels unidirectional?
+        self._unidirectional = True
```

Comment:
Maybe an init or kwarg param.

Reply:
I've removed this entirely from the SBIG filterwheels now.